lib/Infer/ExhaustiveSynthesis.cpp

```diff
@@ -51,9 +53,9 @@ namespace {
   static cl::opt<bool> EnableBigQuery("souper-exhaustive-synthesis-enable-big-query",
     cl::desc("Enable big query in exhaustive synthesis (default=false)"),
     cl::init(false));
-  static cl::opt<bool> UseAlive("souper-use-alive",
-    cl::desc("Use Alive2 as the backend"),
-    cl::init(false));
+  static cl::opt<bool, true>
```

Comment:
please add a /* foo=*/ comment here saying what the true is for

Reply:
Done